ONSdigital/ras-frontstage
# created redirect to maintenance page

# What and why?
The purpose of this PR is to demonstrate a prototype for the maintenance page redirect.

# How to test?
Change the UNDER_MAINTENANCE variable in config.py to True and then access frontstage to check that it redirects to the maintenance page template.

# Trello
[Card](https://trello.com/c/0lGBGkDQ)
